website/source/layouts/docs.erb

```diff
@@ -573,6 +573,17 @@
         <ul class="nav">
           <li<%= sidebar_current("docs-platform-k8s-run") %>>
             <a href="/docs/platform/k8s/run.html">Running Consul</a>
+              <ul class="nav">
```

Comment:
The sidebar doesn't stay expanded when we click on these subpages

Reply:
@alvin-huang Do you know how to fix this?

Reply:
side nav should be fixed in the latest commit